pyproject.toml

```diff
@@ -175,4 +175,4 @@ quiet = false
 
 [tool.codespell]
 skip = '*.ts,*.lock,*package-lock.json'
-ignore-words-list = "installationa, mke" # 'InstallationA' and 'Missing key error'
\ No newline at end of file
+ignore-words-list = "installationa, mke" # 'InstallationA' and 'Missing key error', ellipsis
```

Comment:
forgot to add "..." to list?

Reply:
Yes, added in the comment but spellchecker passed after and not before. Will add the word in the actual list.